frontend/zc_messaging/package.json

```diff
@@ -49,8 +49,11 @@
     "@emotion/styled": "11",
     "@reduxjs/toolkit": "^1.6.2",
     "axios": "^0.21.4",
+    "bootstrap": "^5.2.3",
```

Comment:
don't introduce redux so we don't have inconsistencies in how the app is being styled

Reply:
i meant bootstrap
my bad

Reply:
it looks like there's no style library though
I'll resolve